test/render-code.js

```diff
@@ -0,0 +1,35 @@
+const test = require('tape');
+const renderCode = require('../lib/render-code');
+const cheerio = require('cheerio');
+const code = getCode();
+
+test('should render code with javascript language', t => {
+    const snippet = renderCode(code, 'javascript');
+    const $ = cheerio.load(snippet);
+
+    t.equal($('pre').attr('class'), 'language-javascript');
+    t.notEqual($('span').length, 0);
```

Comment:
I think it's more important there are `.token`s, than `span`s, and that they're are one or more of them, so:
```javascript
t.equal($('.token').length >= 1, true);
```